src/time-ago.pipe.ts

```diff
@@ -61,16 +61,16 @@ export class TimeAgoPipe implements PipeTransform, OnDestroy {
     });
   }
 
-
-  private removeTimer() {
+  private removeTimer(): void {
```

Comment:
why add return type? TS should automatically infer this for us

Reply:
An explicit return type isn't technically necessary, but to me it makes the code clearer to read. A couple of other methods in this class already had return types, so it seems consistent to make this one explicit as well.

Let me know if this change is okay, or if I should revert it before the pull request can be merged.

Reply:
Yes, please revert this one. Thank you!

Reply:
Thx!